setup.cfg

```
@@ -1,2 +1,39 @@
-[wheel]
-universal=1
+[metadata]
+name = incremental
+version = attr: incremental._setuptools_version
+maintainer = Amber Brown
+maintainer_email = hawkowl@twistedmatrix.com
+url = https://github.com/twisted/incremental
+classifiers =
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
```

Based on https://github.com/twisted/incremental/pull/70 we can add a few.

```suggestion
    Programming Language :: Python :: 3.6
    Programming Language :: Python :: 3.7
    Programming Language :: Python :: 3.8
    Programming Language :: Python :: 3.9
```